StompDestination.php

```diff
@@ -65,6 +76,10 @@ public function getQueueName(): string
             throw new \LogicException('Destination name is not set');
         }
 
+        if ($this->extensionType === ExtensionType::ARTEMIS) {
+            return $this->getStompName();
+        }
```

Comment:
A fix for the initial issue, names in artemis don't have to conform to any specific hierarchy or syntax.